examples/pytorch/msn.py

```diff
@@ -84,6 +84,9 @@ def forward_masked(self, images):
 )
 
 criterion = MSNLoss()
+# Or for PMSN:
```

Comment:
I'd create separate examples as we did before. If someone is interested in PMSN they can find it under examples and use it right away. It also shows exactly that everything except the loss is the same. We did the same for NNCLR and other models.